idseq_dag/util/m8.py

```diff
@@ -216,7 +216,7 @@ def read_file_into_set(file_name):
 
 @command.run_in_subprocess
 def call_hits_m8(input_m8, lineage_map_path, accession2taxid_dict_path,
-                 output_m8, output_summary, min_alignment_length, taxon_blacklist=None, taxon_whitelist=None):
+                 output_m8, output_summary, min_alignment_length, taxon_blacklist=None):
```

Comment:
I understand if it is out of scope for this commit, but it would be really nice to move the taxon_blacklist as well, since it suffers from the same issue as the whitelist. They are very similar and it's confusing for them to affect different parts of the pipeline.

Reply:
Assuming that is okay for blacklist items to appear in the raw files (similar to our discussion for the whitelist items)

Reply:
Ok, I have made the equivalent change for the blacklist